examples/dev_flow/debug/debugging_visual.rst

```diff
@@ -0,0 +1,72 @@
+
+.. _examples_dev_flow_debug_visual:
+
+
+Debugging shared libraries with Visual Studio
+=============================================
+
+Conan packages don't contain the necessary information for debugging libraries with Visual Studio by default.
```

Comment:
I think it would be start with a reference to the previous example on how to debug dependencies here: https://docs.conan.io/2/examples/dev_flow/debug/step_into_dependencies.html